index.html

```diff
@@ -103,6 +117,7 @@
           Start Game
         </mdl-button>
         <div id="main-view">
+           <papan-user-profile id="profile-view" style="display: none"/>
         </div>
       </mdl-content>
     </mdl-layout>
```

Comment:
If all this got wrapped into a Polymer component, I think it will be possible to use databinding to do the styles updates (either through binding on attributes or through binding on the style (class$="new-styles")). What do you think of this idea?